Dockerfile

```diff
@@ -4,6 +4,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get dist-upgrade -y &&
 
 # also include bundler 1
 RUN gem install bundler -v '~> 1.0'
+RUN gem install bundler -v '~> 2.0'
```

Comment:
This is already installed by the `apt-get install bundler` line above, because Ubuntu 20.04 already uses 2 by default (https://packages.ubuntu.com/search?keywords=bundler&searchon=names&suite=focal&section=all). That is why we had to manually install version 1 here.

Reply:
Its mostly caused by a bad bundler version in the apt repository: https://github.com/erikhuda/thor/issues/721. Bundler installed with Gem does not have this issue. We could remove the `apt-get install bundler` from the Dockerfile though.